docs/system/system-metrics.md

```diff
@@ -1464,6 +1465,58 @@ See also `MemAvailable` in [/proc/meminfo](https://man7.org/linux/man-pages/man5
 
 
 
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+### Metric: `system.linux.memory.slab.usage`
```

Comment:
currently we have both of these namespaces in this PR:

* `linux.*`
* and `system.linux.*`

should we rename `system.linux.*` to `linux.*`?

that would seem to align with our decision to move `process.runtime.jvm.*` to `jvm.*`

Reply:
I am not sure about this one, the idea of these metric was to fix the current structure of "system.memory.*" metrics for Linux only ones. First step, was to add the `system.linux.memory.available` metric, and secondly, move the slab metric into the "linux" namespace too. I think it is still important to differentiate from where this metric is extracted, whether it is the system or a process. How would this play with the current system metrics? How could I indicate that the metric is related to the system if we remove the "system" namespace?

Reply:
Is slab memory usage reported per-process? Would it be reported as a container metric?
If it's highly unlikely that we'd add it to other namespaces, I'd suggest to define it in the root `linux` namespace.

Also curious what @open-telemetry/semconv-container-approvers think.

I think it's a part of a bigger discussion on why we have so many similar metrics - 
https://github.com/open-telemetry/semantic-conventions/issues/1161

Reply:
I don't think the Docker Stats API provides such a metric: https://docs.docker.com/engine/api/v1.45/#tag/Container/operation/ContainerStats
`/stats/summary` that `kubeletstats` receiver scrapes from does not provide that as well from what I can see: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/kubeletstatsreceiver/testdata/stats-summary.json

Reply:
I believe this PR does not change the status-quo on how `system.{os}` metrics are reported  - https://github.com/open-telemetry/semantic-conventions/blob/main/docs/system/system-metrics.md#systemos---os-specific-system-metrics 

so I'm going to resolve this, thanks for clarifications

Reply:
Also created https://github.com/open-telemetry/semantic-conventions/issues/1255 to discuss `system.linux` vs `linux`